pkg/printers/internalversion/describe.go

```diff
@@ -3332,7 +3333,9 @@ func describePodSecurityPolicy(psp *extensions.PodSecurityPolicy) (string, error
 		w.Write(LEVEL_1, "Required Drop Capabilities:\t%s\n", capsToString(psp.Spec.RequiredDropCapabilities))
 		w.Write(LEVEL_1, "Allowed Capabilities:\t%s\n", capsToString(psp.Spec.AllowedCapabilities))
 		w.Write(LEVEL_1, "Allowed Volume Types:\t%s\n", fsTypeToString(psp.Spec.Volumes))
-
+		if psputil.PSPAllowsFSType(psp, extensions.FlexVolume) {
```

Comment:
no need for additional import and PSPAllowsFSType check, just print this `if len(psp.Spec.AllowedFlexVolumes) > 0`